editors-definitions/che-code-server-latest.yaml

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright (c) 2025 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+schemaVersion: 2.2.2
+metadata:
+  name: che-code-server
+  displayName: Visual Studio Code Server
+  description: Visual Studio Code server for Eclipse Che - latest
```

Comment:
do we need to mention Eclipse Che here? would there be more follow up with downstreaming?

Reply:
We have to add a separate editor definition for downstream

Reply:
Eclipse Che is mentioned in every editor definition.
It is not a problem to remove it, but then I suppose we have to update all the definitions.